src/main/java/engine/property/manager/PropertyManager.java

```diff
@@ -46,11 +50,27 @@ public static BrowserName getBrowserName() {
         return BrowserName.valueOf(getProperty(PropertyKeys.BROWSER_NAME));
     }
 
+    public static Currency getCurrency() {
+        return Currency.valueOf(getProperty(PropertyKeys.COOKIE_currency));
+    }
+
+    public static Set<Cookie> getCookiesFromProperty() {
+        return properties.stringPropertyNames().stream()
+                .filter(propertyName -> propertyName.startsWith("COOKIE_"))
```

Comment:
Maybe use CURRENCY_COOKIE_NAME instead all of this magic spells?:)

Reply:
I simplified the implementation of setting cookies from properties because the application does not set many cookies, so I decided not to search the entire file for prefixes (COOKIE_), and just search for the specific key.
```java
    public static Cookie getCurrencyCookieFromProperties() {
        return new Cookie("currency", PropertyKeys.COOKIE_CURRENCY.toString());
    }
```